shell/server/sandcats.js

```diff
@@ -0,0 +1,100 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2014 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var querystring = Npm.require("querystring");
+var https = Npm.require("https");
+var fs = Npm.require("fs");
+var dgram = Npm.require("dgram");
+var Url = Npm.require("url");
+
+var SANDCATS_HOSTNAME = Meteor.settings && Meteor.settings.public &&
+                          Meteor.settings.public.sandcatsHostname;
+var SANDSTORM_ALTHOME = Meteor.settings && Meteor.settings.home;
+var SANDCATS_VARDIR = (SANDSTORM_ALTHOME || "") + "/var/sandcats";
```

Comment:
Actually, doesn't proxy.js already expose these as globals? I don't think you need to copy them here.